script.js

```diff
@@ -2,19 +2,4 @@ function sum ()
 {var a = Number (document.getElementById ("numberinput1").value);
 var b = Number (document.getElementById ("numberinput2").value);
 var answersum = document.getElementById ("answersum").innerHTML = a + b;
-} 
-
-function subtract ()
-{var c = Number (document.getElementById ("numberinput3").value);
-var d = Number (document.getElementById ("numberinput4").value);
-var answersubtract = document.getElementById ("answersubtract").innerHTML = c - d;}
-
-function multiply ()
-{var e = Number (document.getElementById ("numberinput5").value);
```

Comment:
I see you tried to do all methods here. but that is part of web calculator part 2. Dont worry, wont impact grading.